pallene/driver.lua

```diff
@@ -60,14 +60,18 @@ function driver.compile_internal(filename, input, stop_after)
         return module, errs
     end
 
-    module, errs = uninitialized.verify_variables(module)
-    if stop_after == "uninitialized" or not module then
-        return module, errs
+    if stop_after ~= "optimize" or opt_passes["uninitialized"] then
+        module, errs = uninitialized.verify_variables(module)
+        if stop_after == "uninitialized" or not module then
+            return module, errs
+        end
```

Comment:
The "uninitialized" pass is not an optimization. It is an analysis pass to see if there are any variables that are used before they are initialized. Therefore, it should not be optional.